modules/aggregations/src/yamlRestTest/resources/rest-api-spec/test/aggregations/time_series.yml

```diff
@@ -76,7 +76,7 @@ setup:
   - match: { hits.total.value: 1 }
   - length: { aggregations: 1 }
 
-  - match: { aggregations.ts.buckets.0.key: { _tsid: "RGqc4j2v9houOhyLbojV0RSqi9yhwd0OzmU" } }
+  - match: { aggregations.ts.buckets.0.key: { _tsid: "vD4dAfOH-KDNoywVO13gvAAAAAAAAAAAAAAAAAAAAACv9houG2TKanX0IDDfq7mPr0SxJA" } }
```

Comment:
I'm wondering whether there's a way not to specify specific `_tsid`s in the tests. For example by just checking that the _tsid field is present.

This avoids having to change a bunch of tests every time we make adjustments to the `_tsid` format. Testing that should be isolated to the `TimeSeriesIdFieldMapperTests` and the  rest of the test shouldn't have a hard-coded assumption about the format and structure of the `_tsid`.

Reply:
Some aggregations are done on the _tsid field...my opinion is that this should not really be exposed outside but just used internally. For now I will keep it because it is useful for me to see that different buckets are created, at least in some cases.